.gitignore

```diff
@@ -0,0 +1,10 @@
+/__src/.sculpin/
+/.idea/
+/__src/app/cache/*
```

Comment:
because top level directory holds page content itself. Also we have `_docs` there. Since there might be some more directories prefixed with single floor `_` I wanted to explicitly split sources from output so nobody confuses generated content from sources. If you think there is a better way to distinguish sources I'm open to changes